src/Cards/AmericanExpress.php

```diff
@@ -46,7 +46,7 @@ class AmericanExpress extends Card implements CreditCard
      *
      * @var array
      */
-    protected $cvc_length = [3, 4];
+    protected $cvc_length = [4];
```

Comment:
But American Express supports 3 and 4 digits:
https://www.americanexpress.com/ca/en/articles/life-with-amex/learn/what-is-credit-card-cvv/

> This **three**- or **four**-digit code tells the merchant that you are in possession of the card

Reply:
I think it's referring to a CVV code in general, since in the previous paragraph it says:

> Depending on your credit card provider [...]

But later, they explicitly say (the part I quoted in the PR):

> If you have an American Express® Card, you will find the **four-digit** credit card CVV [...]